addons/mail/tests/common.py

```diff
@@ -120,3 +120,34 @@ def tearDownClass(cls):
         cls.env['ir.mail_server']._revert_method('build_email')
         cls.env['ir.mail_server']._revert_method('send_email')
         super(TestMail, cls).tearDownClass()
+
+    def gateway_reply_wrecord(self, template, record, extra_irt=True):
```

Comment:
Rename `extra_irt` to `in_reply_to` ? Name is a bit cryptic.

Reply:
Renamed ot use_in_reply_to